tests/test_spark_operator.py

```diff
@@ -0,0 +1,73 @@
+"""
+To run these tests you need to have at least configured airflow in
+local mode and run:
+
+`airflow initdb`
+"""
+import os
+from datetime import datetime
+
+import pytest
+import logging
+from domino.airflow import DominoSparkOperator 
+from domino.exceptions import RunFailedException
+
+TEST_PROJECT = os.environ.get("DOMINO_SPARK_TEST_PROJECT")
+SPARK_ENVIRONMENT_ID = os.environ.get('DOMINO_SPARK_TEST_ENVIRONMENT_ID')
+
+
+def test_spark_operator_no_cluster():
+    airflow = pytest.importorskip("airflow")
+
+    from airflow import DAG
+    from airflow.models import TaskInstance
+
+    dag = DAG(dag_id="foo", start_date=datetime.now())
+    task = DominoSparkOperator(
+        dag=dag,
+        task_id="foo",
+        project=TEST_PROJECT,
+        command="test_spark.py",
```

Comment:
Hey @wseaton can we have python/bash commands instead of name of the test files `test_spark.py`, `test_spark_fail.sh`?

Reply:
Not here, since I can't provide a direct command via the v4 api, before I could do this via `isDirect`. I'd also like to not have to use wrapper scripts :slightly_smiling_face:

Reply:
Hey @wseaton can you add these sample test files (test_spark.py, test_spark_fail.sh) here, in this folder?

Reply:
@abhijeet2096 this has been done